File: [1] BEGINNER/1019 - Conversão de Tempo.py
'''
Leia um valor inteiro, que é o tempo de duração em segundos de um determinado evento em uma fábrica, e informe-o expresso no formato horas:minutos:segundos.

Entrada
O arquivo de entrada contém um valor inteiro N.

Saída
Imprima o tempo lido no arquivo de entrada (segundos), convertido para horas:minutos:segundos, conforme exemplo fornecido.
'''

h = m = s = 0
tempo = int(input())
#While pra calcular as horas:
while True:
    if tempo < 3600:
        break
    else:
        tempo -= 3600
        h += 1
#While para calcular os minutos
while True:
    if tempo < 60:
        break
    else:
        tempo -= 60
        m += 1
s = tempo
print(h,m,s, sep=':')        

File: [1] BEGINNER/1019 - Conversão de Tempo.c
/*
Leia um valor inteiro, que é o tempo de duração em segundos de um determinado evento em uma fábrica, e informe-o expresso no formato horas:minutos:segundos.

Entrada
O arquivo de entrada contém um valor inteiro N.

Saída
Imprima o tempo lido no arquivo de entrada (segundos), convertido para horas:minutos:segundos, conforme exemplo fornecido.
*/

#include <stdio.h>
 
int main() {
    int t, h, m, s; h=m=s=0;
    
    scanf("%d", &t);
    s = t;
    while(s >= 60){
        s -= 60; m++;
    }
    while(m >= 60){
        m -= 60; h++;
    }
    printf("%d:%d:%d\n", h, m, s);
 
    return 0;
}